spark-plugins/src/main/java/co/cask/hydrator/plugin/spark/SparkUtils.java

```diff
@@ -84,7 +85,7 @@ static void validateConfigParameters(Schema inputSchema, @Nullable String featur
       throw new IllegalArgumentException("Cannot specify values for both featuresToInclude and featuresToExclude. " +
                                            "Please specify fields for one.");
     }
-    Map<String, Integer> fields = new HashMap<>();
+    Map<String, Integer> fields = new LinkedHashMap<>();
```

Comment:
I see. This is a util class and usage of the method `getFeatureList()` does not depend on insertion order of fields. So I think it would be better if we make changes into `DecisionTreeConfigTest` rather than into `SparkUtils`